tests/framework/installer/ceph_installer.go

```diff
@@ -45,7 +45,9 @@ const (
 	// test with the latest nautilus build
 	nautilusTestImage = "ceph/ceph:v14"
 	// test with the latest octopus build
-	octopusTestImage  = "ceph/ceph:v15"
```

Comment:
Would be nice to split into two commits... one for the changes to existing tests and one for the new test

Reply:
#5269 fir changes that affect other tests. When this will be merged i will rebase on it.